lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -877,8 +877,11 @@ LogicalResult rewrite0DBinaryTensorOp(Operation *op,
     return rewriter.notifyMatchFailure(
         op, "only int scalar lhs or rhs is supported");
   }
-  if (isa<AtenSubTensorOp, AtenSubScalarOp, AtenAddTensorOp, AtenAddScalarOp>(
-          op)) {
+  if (isa<AtenRsubScalarOp>(op)) {
+    std::swap(lhs, rhs);
```

Comment:
nit: I would prefer being more explicit here to avoid confusion as to what `lhs` and `rhs` represent. 

Here I would instead insert an `if` inside the `if` with all the `sub` and `add` ops:

```c++
if (isa<AtenSubTensorOp, ...>(op)) {
  if (isa<AtenRsubScalarOp>(op))
    rhs = rewriter.create<AtenMulIntOp>(loc, lhs, alpha);
  else
    rhs = rewriter.create<AtenMulIntOp>(loc, rhs, alpha);
}
```

And below where `AtenSubIntOp` gets created, I would handle `AtenRsubScalarOp` on its own.

This is a bit more verbose, but it makes sure someone reading the code does not need context from different places to understand how things work.